threeML/plugins/HAWCLike.py

```diff
@@ -546,6 +546,13 @@ def display_residuals_at_position(self, ra, dec, radius=0.5, pulls=False):
 
         return fig
 
+    def get_number_of_data_points( self ):
+        """
+        Number of data point = number of pixels.
+        Implemented in liff as the number of pixels in the ROI per analysis bin.
+        """        
+        pixels_per_bin =  np.array( self._theLikeHAWC.GetNumberOfPixels() )
+        return np.sum( pixels_per_bin )
```

Comment:
If this is always going to be an integer, can you make it explicit by casting to int?

```python
return int(np.sum( pixels_per_bin ))
```